api/scripts/certification/rescore-certifications.js

```diff
@@ -0,0 +1,43 @@
+import 'dotenv/config';
+
+import * as url from 'node:url';
+
+import { disconnect, knex } from '../../db/knex-database-connection.js';
+import { CertificationRescoringByScriptJob } from '../../lib/infrastructure/jobs/certification/CertificationRescoringByScriptJob.js';
+import { logger } from '../../src/shared/infrastructure/utils/logger.js';
+
+const modulePath = url.fileURLToPath(import.meta.url);
+const isLaunchedFromCommandLine = process.argv[1] === modulePath;
+
+async function main(certificationCourseIds) {
+  await _scheduleRescoringJobs(certificationCourseIds);
+}
+
+(async () => {
+  if (isLaunchedFromCommandLine) {
+    try {
+      const certificationCourseIds = process.argv[2].split(',');
+      await main(certificationCourseIds);
```

Comment:
Si gestion d'erreur (voir commentaire plus haut)

```suggestion
      const exitCode = await main(certificationCourseIds);
      return exitCode;
```